File: models/objects/parrots.js
var mongoose = require('mongoose'),
	Schema = mongoose.Schema,
	_ = require('underscore'),
	returnJSON = require('./utils').returnJSON;

var parrots_schema = new Schema({
	twitter_id: String,
	oauth_token: String,
	oauth_token_secret: String,
	created_at: {type: Date, default: Date.now},
	twitter_info: {},
	payments: [],
	id: String
});

mongoose.model('Parrots', parrots_schema);
module.exports = Parrots = mongoose.model('Parrots');

Parrots.prototype.returnJSON = returnJSON;

File: dal/payparrot_dal/parrots.py
from datetime import datetime
from payparrot_dal.base import BaseModel

class Parrots(BaseModel):
    _meta = {
        'collection': 'parrots',
        'fields': {
            'twitter_id': {'required': True},
			'oauth_token': {'required': True},
			'oauth_token_secret': {'required': True},
			'created_at': {'default': datetime.now, 'readonly': True, 'private': True},
			'twitter_info': {'default': {}},
			'payments': {'default': []},
    	}
    }